blueprints/app/files/package.json

```diff
@@ -12,6 +12,8 @@
   "repository": "",
   "scripts": {
     "build": "ember build",
+    "lint": "eslint *.js addon app config lib node-tests server tests",
+    "lint:fix": "eslint --fix *.js addon app config lib node-tests server tests",
```

Comment:
Can this be `npm run lint -- --fix`? Or would yarn hate that? Maybe we can sub in yarn here too.

Reply:
I generally find `npm run lint -- --fix` to be very hard to read (due to repeated `--` with different meaning)...

Are you mostly referring to removing the duplication between `lint` and `lint:fix`?

Reply:
yep the duplication.